.github/workflows/elixir.yml

```diff
@@ -12,12 +12,19 @@ jobs:
   unit_test:
     name: Unit Tests - Elixir ${{matrix.elixir}} / OTP ${{matrix.otp}}
     runs-on: ubuntu-20.04
+    continue-on-error: true
```

Comment:
@tmaszk  👋  Could I ask you about this setting? I'm not a Github Actions expert and AFAICT, this might mean the test run passes even if there are errors? Or maybe this behaves differently than what I'm expecting.   Could you share your reasoning for adding this?

Reply:
@tmaszk  Thanks! A nice confirmation happened as well with [this test run](https://github.com/newrelic/elixir_agent/actions/runs/7611561532/job/21114894502?pr=424).